packages/react-components/src/organisms/ToolModal.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { useState } from 'react';
+import { TeamTool } from '@asap-hub/model';
+
+import ModalEditHeader from '../molecules/ModalEditHeader';
+import { Modal, LabeledTextField } from '../molecules';
+import { noop } from '../utils';
+
+type ToolModalProps = Partial<TeamTool> & {
+  onSave?: (data: TeamTool) => void;
+  title: string;
+  backHref: string;
+};
+
+const ToolModal: React.FC<ToolModalProps> = ({
+  title,
+  url = '',
+  description = '',
+  name = '',
+  onSave = noop,
+  backHref,
+}) => {
+  const [newUrl, setNewUrl] = useState(url);
```

Comment:
I suppose this is one of those instances where copying props to state is correct since you want a snapshot at the time of opening the modal?